third_party/nvfuser/csrc/ir_internal_nodes.h

```diff
@@ -1465,6 +1465,14 @@ class TORCH_CUDA_CU_API IterDomain : public Val {
       bool inner_split,
       bool trim_out_of_bounds);
 
+  //! Resize an IterDomain by expanding both the left and right sides
+  //! by given widths. Optionally mark it as an rfactor domain
```

Comment:
Maybe mention that `left_expansion` and `right_expansion` may be negative, and in this case, will shrink instead of expand.